core-api/src/main/java/com/sequenceiq/distrox/api/v1/distrox/model/DistroXV1Request.java

```diff
@@ -60,6 +60,8 @@ public class DistroXV1Request extends DistroXV1Base implements TaggableRequest {
 
     private Integer javaVersion;
 
+    private boolean multiAz;
```

Comment:
I would rename it to `enableMultipleAvailabilityZones` or something more trivial for end users.